src/sqlancer/datafusion/server/datafusion_server/src/main.rs

```diff
@@ -0,0 +1,463 @@
+#[cfg(feature = "datafusion_dev")]
```

Comment:
Would it be possible to add a one-line comment to explain the purpose of this file?